changelog/unreleased/pull-3991

```diff
@@ -0,0 +1,5 @@
+Enhancement: Adds progress details while restoring
+
+Example: [0:42] 23 / 3456 Files, 12.98 MiB / 23.54 GiB, 5.76 %
```

Comment:
This can do with some more prose:

```suggestion
The command restic restore now shows a progress report while it's restoring files.

Example: [0:42] 23 / 3456 Files, 12.98 MiB / 23.54 GiB, 5.76 %
```